src/screens/OtherProfile/index.js

```diff
@@ -603,7 +524,8 @@ const OtherProfile = () => {
   };
 
   const handleDataFeed = () => {
-    const isFeedsShown = !blockStatus.blocked && !blockStatus.blocker;
+    if (isProfileFetching) return [];
+    const isFeedsShown = !(isBlocking || isBlocked);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditions `if (isBlocking)` and `if (isBlocked)` replace the previous `if (blockStatus.blocker)` and `if (blockStatus.blocked)`. Ensure that `isBlocking` and `isBlocked` are equivalent to `blockStatus.blocker` and `blockStatus.blocked` respectively to avoid any behavioral changes.



<!-- This is an auto-generated comment by OSS CodeRabbit -->